frontend/catalyst/from_plxpr.py

```diff
@@ -305,7 +305,7 @@ def interpret_operator_eqn(self, eqn: jax.core.JaxprEqn) -> None:
                 *wires,
                 *invals,
                 op=eqn.primitive.name,
-                params_len=len(eqn.invars) - eqn.params["n_wires"],
+                ctrl_value_len=0,
```

Comment:
the reason for this change is that after using using bind_flexible_primitive, the gate parameters will be appended to the list of all dynamic parameters and hence it will change the order of arguments. previously it was [qubits, params, ctrl_qubits, ctrl_values], but after the change it is [qubits, ctrl_qubits, ctrl_values, params]. so we need to know the length of ctrl_values instead of params.

Reply:
Oh I see, the plxr converter doesn't support dynamic control qubits yet, so the 0 makes sense.

I just realized though that `ctrl_len` and `ctrl_value_len` should always be the same, so we might not need this variable and/or assert that it they are the same.

Reply:
Still feel weird about the `ctrl_value_len` parameter being here, semantically it just feels out of place / not that meaningful. Can we just get rid of it?

Reply:
oh I just remembered that I added this to pass a test. There are some tests in `frontend/test/pytest/test_from_plxpr.py` that compare the jaxpr from plpxr and catalyst. If I remove the `ctrl_value_len` then that test fails because one will have the `ctrl_value_len` in the jaxpr and the other will not. But otherwise I agree that there is not point of having it there.